### PR TITLE
Add support for schema 27

### PR DESCRIPTION
**Pull request by raman325**
See the following PRs:
- https://github.com/zwave-js/zwave-js-server/pull/892
- https://github.com/zwave-js/zwave-js-server/pull/893

### CODE REVIEW

File: zwave_js_server/model/controller/statistics.py [hunk 1]
```diff
@@ -42,19 +42,85 @@ def nlwr(self) -> Optional[RouteStatistics]:
...
+    average: int
+    current: int
+
+
+class BackgroundRSSIDataType(TypedDict, total=False):
+    """Represent a background RSSI data dict type."""
+
+    # https://github.com/zwave-js/node-zwave-js/blob/master/packages/zwave-js/src/lib/controller/ControllerStatistics.ts#L40
+    timestamp: int  # required
+    channel0: ChannelRSSIDataType  # required
```
Comment (MartinHjelmare) [inline zwave_js_server/model/controller/statistics.py:57]: Let's use `Required` from `typing_extensions`

File: zwave_js_server/model/controller/statistics.py [hunk 2]
```diff
@@ -42,19 +42,85 @@ def nlwr(self) -> Optional[RouteStatistics]:
...
+    messagesDroppedRX: int  # required
+    NAK: int  # required
+    CAN: int  # required
+    timeoutACK: int  # required
+    timeoutResponse: int  # required
+    timeoutCallback: int  # required
+    backgroundRSSI: BackgroundRSSIDataType
+
+
+class ChannelRSSI:
```
Comment (MartinHjelmare) [inline zwave_js_server/model/controller/statistics.py:78]: Alternatively make a dataclass, if we're not going to update the data after instantiation.

```py
@dataclass
class ChannelRSSI:
    data: ChannelRSSIDataType
    average: int = field(init=False)
    current: int = field(init=False)

    def __post_init__(self) -> None:
        self.average = self.data["average"]
        self.current = self.data["current"]
```

File: zwave_js_server/model/node/__init__.py [hunk 3]
```diff
@@ -755,6 +755,20 @@ async def async_interview(self) -> None:
             require_schema=22,
         )
 
+    async def async_get_value_timestamp(self, val: Union[Value, str]) -> int:
```
Comment (MartinHjelmare) [inline zwave_js_server/model/node/__init__.py:758]: Update the union syntax if merged after the other PR.

File: zwave_js_server/model/value.py [hunk 4]
```diff
@@ -153,6 +155,16 @@ def value_size(self) -> Optional[int]:
         """Return valueSize."""
         return self.data.get("valueSize")
 
+    @property
+    def stateful(self) -> Optional[bool]:
```
Comment (MartinHjelmare) [inline zwave_js_server/model/value.py:159]: Update union syntax.

File: zwave_js_server/model/controller/statistics.py (hunk 2 above)
Comment (raman325) [reply zwave_js_server/model/controller/statistics.py:78]: good idea

File: zwave_js_server/model/controller/statistics.py (hunk 1 above)
Comment (raman325) [reply zwave_js_server/model/controller/statistics.py:49]: I am planning to come back and do this all as part of https://github.com/home-assistant-libs/zwave-js-server-python/issues/590 where I will review the interfaces across zwave-js, zwave-js-server, and the lib

Comment (MartinHjelmare) [reply zwave_js_server/model/controller/statistics.py:49]: Ok.

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**